### DISCUSSION TITLE
Ask for a locker and get TC as a syndie

### DISCUSSION THREAD

**Discussion by BlitzGunner225** [unitystation/unitystation#8369] [Ideas]
The rough idea is that you can get more TC if you spawn your syndie items in a syndie locker.
You would have to hide or space this locker as otherwise someone will find it and know that there is a syndie agent aboard.
Mechanically, you would ask for a locker, you would put in at least 8 TC of items in the ui, and then you would get a locker with your items and one free TC.

**Comment by CdrCross** [comment]
Destroying a locker with a welding tool or yeeting it out into the void would be trivial.

**Reply by BlitzGunner225** [reply]
1. Oh shit I didn’t know that. Maybe it could be undestroyable or it could be made of a special syndie metal so you still have to hide some stuff.
2. I don’t think it’s easy to space stuff on most maps as some have Eva suits locked up, might cams on the space exits, and you usually have to go through populated areas.

**Reply by PerfectTangent** [reply]
spacing things really is trivial. almost a third of the entire roster in the game already come with space suits as part of their job, and once it was found, it'd be in the best interest of everyone except the syndicate operative to destroy, space, or secure the locker somewhere. That doesn't neuter the concept in my eyes, however.

**Comment by PerfectTangent** [comment]
what is the point of the idea? I get that it's simulating a stash or hideaway of stuff, and the bonus TC is to incentivize using it, but it's unclear to me what the goal of the feature is. Could you explain further?

**Reply by BlitzGunner225** [reply]
You in essence get more TC in trade of another light objective you have to take care of.
You don’t technically need to deal with it, but that just means someone will likely find it and you’ll loose your element of surprise.
That’s at least what I want it to be.

**Reply by AtnerNT** [reply]
The main problem i have this idea rn is this: A station is already in a state of paranoia- wether through alert level changes or centocmm notifications. The station is usually always aware that a traitor threat is imminent. A person finding a locker in maint doesnt really do anything- the locker cant be traced to the specific person, it just tells the crew "yeah theres a traitor" which they already know. If the locker has someway of tracing it back to the specific syndie- it is far from perfect but I think its an improvement- It places far more value on finding a locker, and far more pressure to hide it properly. But this still traces back to the problem that a locker is pretty easy to just space and never worry about again. I think this idea has potential- but just needs some kinks worked out

**Comment by BlitzGunner225** [comment]
Ok so what do you all think about this addition 
1. The locker will be indestructible, but also made of a special metal that explodes when there is an absence of air (like space). 
This means that you have to either hide the locker in maintenance (the riskier but easier approach) or throw it out of a moving shuttle (the safer but harder approach). 
2. The first spawned locker has the name of the user on it. The second has the coordinates of the user that updates every minute. 
This makes it so you shouldn’t hide both lockers together and that buying two isn’t the same as buying one. It also means that a found locker means a lot more trouble.
